frontend/src/variants/components/FilterForm.vue

```diff
@@ -58,6 +58,30 @@ const showOverlay = computed(() =>
   ['initial', 'initializing'].includes(variantQueryStore.storeState),
 )
 
+var eventMethod = window.addEventListener ? 'addEventListener' : 'attachEvent'
+var eventer = window[eventMethod]
+var messageEvent = eventMethod == 'attachEvent' ? 'onmessage' : 'message'
+var imageRes
+
+// Listen to message from child window
+eventer(
+  messageEvent,
+  function (event) {
+    var key = event.message ? 'jsonRes' : 'data'
+    imageRes = event[key]
+
+    if (event.origin == 'http://127.0.0.1:7000') {
```

Comment:
@ahujameg A hard coded address seems wrong to me. Is this ensured to work on other instances as well? Urls/adresses should be defined elsewhere.

Reply:
Now defined the URL for the middleware application in the settings (base.py).